include/RAJA/exec-cuda/scan_cuda.hxx

```diff
@@ -103,6 +103,16 @@ struct minimum : public ::thrust::minimum<T> {
 
 }  // closing brace for cuda_scan namespace
 
+namespace scan
+{
+
+template <typename T>
+struct defaults <cuda_exec, T> {
```

Comment:
I pulled this out later in my commits. Ideally, I think something along the lines of:

`RAJA::operators::gnu` and `RAJA::operators::cpu` might be nice. We could also get away with a bunch of `using ::thrust::plus`; statements for `operators::gnu` and using `::std::plus;` statements for `operators::cpu`